.github/workflows/dev.yml

```diff
@@ -11,3 +11,5 @@ jobs:
     uses: ./.github/workflows/ci.yml
     with:
       mvnArgs: '"-Divy.engine.download.url=https://dev.axonivy.com/permalink/dev/axonivy-engine.zip" "-Divy.engine.version=(10.0.0,]"'
```

Comment:
you'll have to adapt the parameters regarding the '-Dproject.build.plugin.version' from the official pipeline, in order to fix the dev-pipeline.